workflows/sample/sample.trial.wdl

```diff
@@ -84,10 +96,18 @@ workflow sample_trial {
     Array[Array[Array[File]]] affected_person_svsig_gv             = if defined(sample_affected_person.svsig_gv)                    then sample_affected_person.svsig_gv else []
     Array[IndexedData] affected_person_deepvariant_phased_vcf_gz   = if defined(sample_affected_person.deepvariant_phased_vcf_gz)   then sample_affected_person.deepvariant_phased_vcf_gz else 0
     Array[File?] affected_person_jellyfish_output_files            = if defined(sample_affected_person.jellyfish_output)            then sample_affected_person.jellyfish_output else []
+    Array[File?] affected_person_tandem_genotypes                  = if defined(sample_affected_person.tandem_genotypes)            then sample_affected_person.tandem_genotypes else []
+    Array[File?] affected_person_tandem_genotypes_absolute         = if defined(sample_affected_person.tandem_genotypes_absolute)   then sample_affected_person.tandem_genotypes_absolute else []
+    Array[File?] affected_person_tandem_genotypes_plot             = if defined(sample_affected_person.tandem_genotypes_plot)       then sample_affected_person.tandem_genotypes_plot else []
+    Array[File?] affected_person_tandem_genotypes_dropouts         = if defined(sample_affected_person.sample_tandem_genotypes_dropouts) then sample_affected_person.tandem_genotypes_dropouts else [] 
```

Comment:
I'm not sure I understand the difference between `affected_person_*` and `unaffected_person_*` with regards to tandem genotypes.  What is the difference here?

Reply:
so we subdivide the patients in the inputs into two object arrays based on their status- affected and unaffected. fundamentally there's no different at this point- but the operations change when you are affected (ie hifiasm is only called on sample line for affected patients- we don't assemble unaffected individuals). 

But for Cromwell, that means we have to catch the outputs at the end in this manner.

Reply:
```(ie hifiasm is only called on sample line for affected patients- we don't assemble unaffected individuals).```

Is this logic that CMKC asked for?  In our version of the workflow, we generate sample assemblies for every sample, regardless of whether they're unaffected or not.  We only generate trio-binned assemblies for trio kids (again, regardless of whether unaffected or affected).

Reply:
so our understanding was that we can't assume every input will be a singleton or trio. Therefore, we don't assume trio logic in our implementation- ergo the "affected" vs "unaffected" arrays instead.

with this logic, it allows the user to be able to have more granular control over what specifically they want done for each sample. We can absolutely assemble every sample regardless status if that's what's needed, but given the cost/computational burden of the assemblies, we were only looking to assemble what the user specifically wanted assembled- and given our understanding that oftentimes the parental data isn't long reads but illumina data, it didn't make sense to make assembly a default setting for the unaffecteds at the time.

We do still perform logic checks that if a user specifically wants trio-binned assembly, they must have provided a pedigree that presents a trio to do this.